src/Native/Runtime/gcrhenv.cpp

```diff
@@ -1504,6 +1416,159 @@ bool GCToEEInterface::EagerFinalized(Object* obj)
     return false;
 }
 
+bool GCToEEInterface::IsGCThread()
+{
+    return false;
+}
+
+bool GCToEEInterface::WasCurrentThreadCreatedByGC()
+{
+    return false;
+}
```

Comment:
These two methods did not have a specialization in corert so i copied the defaults that were removed from the GC headers.
I think `WasCurrentThreadCreatedByGC` could be implemented as `return ThreadStore::GetCurrentThread()->IsGCSpecial();`?
Not sure about `IsGCThread`

Reply:
Yes, `WasCurrentThreadCreatedByGC ` should be implemented as `IsGCSpecial`.

`IsGCThread` should be `pCurrentThread->IsGCSpecial() || pCurrentThread == ThreadStore::GetSuspendingThread()`